convertToSingleHost.js

```diff
@@ -119,12 +119,20 @@ async function updatePackageJsonForSingleHost(host) {
   await writeFileAsync(packageJson, JSON.stringify(content, null, 2));
 }
 
-async function updateLaunchJsonFile() {
+async function updateLaunchJsonFile(host) {
   // Remove 'Debug Tests' configuration from launch.json
   const launchJson = `.vscode/launch.json`;
   const launchJsonContent = await readFileAsync(launchJson, "utf8");
   const regex = /(.+{\r?\n.*"name": "Debug (?:UI|Unit) Tests",\r?\n(?:.*\r?\n)*?.*},.*\r?\n)/gm;
-  const updatedContent = launchJsonContent.replace(regex, "");
+  var updatedContent = launchJsonContent.replace(regex, "");
+  // Only keep host configuration
+  for (index in hosts) {
```

Comment:
The iterative pattern used here is `hosts.foreach( function (host) {...}) ;`.  We should probably be consistent.

Reply:
Done